pkgs/applications/altcoins/ledger-live-desktop/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+ stdenv,
+ appimage-run,
+ fetchurl,
+ runtimeShell,
+}:
+
+stdenv.mkDerivation rec {
+  github-user = "LedgerHQ";
+  pname = "ledger-live-desktop";
+  version = "1.9.1";
+
+  src = fetchurl {
```

Comment:
Consider using `fetchFromGitHub`, which is more efficient because it fetches just the tar.gz of the given commit (shallow), instead of doing a full git clone and throwing away the `.git` dir.

Reply:
@bhipple but we don't want the tarball of the git repo we want the app image that is built and shipped as part of the releases